emacs-everywhere.el

```diff
@@ -330,12 +349,14 @@ Never paste content when ABORT is non-nil."
         (when (and (frame-parameter nil 'emacs-everywhere-app)
                    emacs-everywhere-paste-command
                    (not abort))
+	  (sleep-for 0.1)
+
           (apply #'call-process (car emacs-everywhere-paste-command)
                  nil nil nil (cdr emacs-everywhere-paste-command)))))
     ;; Clean up after ourselves in case the buffer survives `server-buffer-done'
     ;; (b/c `server-existing-buffer' is non-nil).
     (emacs-everywhere-mode -1)
-    (server-buffer-done (current-buffer))))
+    (server-kill-buffer)))
```

Comment:
Why is this change being made?

Reply:
My personal solution to #70. But yeah, since this have not much to do with the topic (MS-Windows support), I'll revert this.